falcon/response.py

```diff
@@ -136,12 +145,120 @@ def set_stream(self, stream, stream_len):
         self.stream = stream
         self.stream_len = stream_len
 
+    def set_cookie(self, name, value, expires=None, max_age=None,
+                   domain=None, path=None, secure=True, httponly=True):
```

Comment:
Ok. :+1:

You sure though?
The spec defines the flag as `httponly` which is why i choose to reflect that.
In contrast, `max_age` is called `max-age` in the cookie spec.